charms/knative-eventing/metadata.yaml

```diff
@@ -1,23 +1,11 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-# For a complete list of supported options, see:
-# https://juju.is/docs/sdk/metadata-reference
 name: knative-eventing
 description: |
-  TEMPLATE-TODO: fill out the charm's description
+  Knative Eventing is an implementation of serverless functions in a Kubernetes
```

Comment:
Minor nit - ignore if you disagree.  I think this could be `Knative is an implementation of ...` (Eventing isn't really the serverless stuff.  It is important for knative and serverless apps, but if anything Serving is the real serverless stuff).

Reply:
I added the description from the official [docs](https://knative.dev/docs/eventing/)